# Description: MANIFEST.skip is not present in the upstream release, so no way
#              the test for its presence would succeed. File ommitted.
# Author: Damyan Ivanov <dmn@debian.org>
--- a/t/99-manifest.t
+++ b/t/99-manifest.t
@@ -1,12 +1,12 @@
 use strict;
 use Test::More;
 
-# Check that MANIFEST and MANIFEST.skip are sane :
+# Check that MANIFEST is sane
 
 use File::Find;
 use File::Spec;
 
-my @files = qw( MANIFEST MANIFEST.skip );
+my @files = qw( MANIFEST );
 plan tests => scalar @files * 4 +1;
 
 for my $file (@files) {
